Misc/Positions_Area Positions 2 External.txt

```diff
@@ -52,6 +52,8 @@ BIRD_3_CTR:Reykjavik Control:128.600:RK3:C:BIRD:CTR:-:-:4100:4177
 BIRD_4_CTR:Reykjavik Control:126.750:RK4:C:BIRD:CTR:-:-:4100:4177
 BIRD_5_CTR:Reykjavik Control:132.300:RK5:C:BIRD:CTR:-:-:4100:4177
 BICC_FSS:Iceland Radio:127.850:IR:IR:BICC:FSS:-:-:4100:4177
+BICC_A_FSS:Iceland Radio Clearance:129.620:IRD1:IR:BICC:FSS:-:-:4100:4177
```

Comment:
```suggestion
BICC_A_FSS:Iceland Radio Clearance:129.620:IRD1:IR:BICC:FSS:-:-:4100:4177
BICC_A_FSS:Iceland Radio Clearance:129.625:IRD1:IR:BICC:FSS:-:-:4100:4177
```

Reply:
I think it's worth defining both, seeing as during CTP they were using 129.625 as their prim frequency.